src/commands/build.ts

```diff
@@ -44,11 +59,6 @@ export class BuildCommand implements Command {
           'additional `<link rel="prefetch">` tags into ' +
           'entrypoint and `<link rel="import">` tags into fragments and shell.'
     },
-    {
-      name: 'html.collapseWhitespace',
```

Comment:
Yes, see: https://github.com/Polymer/polymer-cli/pull/524/files#diff-6efc8ba30392e1492e440aea247b32f6R155

With the way `command-line-args` works, anyone who is using this will get an error once this is removed. That feels like enough to me, wdyt? Are enough people actually using this?